docs/manual/docs/cmd/aad/sp/sp-get.md

```diff
@@ -15,6 +15,7 @@ Option|Description
 `--help`|output usage information
 `-i, --appId [appId]`|ID of the application for which the service principal should be retrieved
 `-n, --displayName [displayName]`|Display name of the application for which the service principal should be retrieved
+`-o, --objectId [objectId]`|ObjectId of the application for which the service principal should be retrieved
```

Comment:
We can't use `-o` because it's reserved for `output`

Reply:
That is correct . I copied that from `aad approleassigment list` so that makes two places to be fixed. Let me do that.

Reply:
Fixed